web/docs/language/features.md

```diff
@@ -1872,6 +1872,25 @@ Make sure to pass in an object expected by the `QueryClient`'s constructor, as
 explained in
 [_react-query_'s docs](https://tanstack.com/query/v4/docs/react/reference/QueryClient).
 
+## Public static files on the client
```

Comment:
Can they import stuff from there if they want? What can they do with the stuff that ends up there? What can they put there besides favicon.ico?